test/sharness/t0002-docker-image.sh

```diff
@@ -50,7 +50,7 @@ test_expect_success "docker image runs" '
 '
 
 test_expect_success "docker container gateway is up" '
-  pollEndpoint -host=/ip4/127.0.0.1/tcp/8080 -http-url http://localhost:8080/api/v0/version -v -tries 30 -tout 1s
```

Comment:
:thought_balloon: I suspect this is used by various monitoring tools.
Mind updating waterworks-infra in preparation for this removal?

`/api/v0/version` seems to be used in a few places, including pingdom:

https://github.com/search?q=repo%3Aipshipyard%2Fwaterworks-infra+api%2Fv0%2Fversion&type=code

Maybe @ns4plabs knows other places.

Reply:
https://github.com/ipshipyard/waterworks-infra/pull/41